recipes/opencv/4.x/conanfile.py

```diff
@@ -131,6 +133,8 @@ def requirements(self):
             self.requires("quirc/1.1")
         if self.options.get_safe("with_gtk"):
             self.requires("gtk/system")
+        if self.options.with_protobuf:
+            self.requires("protobuf/3.5.2@bincrafters/stable")
```

Comment:
Not allowed, CCI recipes must depend on CCI recipes.

Reply:
Should I first create protobuf/3.5.2 recipe?
Unfortunately, there are too many changes to do in opencv to use a newer version

Reply:
Would it mean regenerate files from .proto files, or more changes?

Reply:
it looks like more changes
[build_log.txt](https://github.com/conan-io/conan-center-index/files/6364620/build_log.txt)

Reply:
Did you try with `PROTOBUF_UPDATE_FILES ON`?

Reply:
How to add a version to conandata.yml?
I haven't found an info how to generate sha256 in here
https://docs.conan.io/en/latest/reference/config_files/conandata.yml.html

Reply:
try first to regenerate files, vcpkg uses latest version of protobuf, so it should work.

Reply:
I am not sure, what you mean... I don't find any 'PROTOBUF_UPDATE_FILES' flag

Reply:
https://github.com/opencv/opencv/search?q=PROTOBUF_UPDATE_FILES

Reply:
So you probably need:
- `BUILD_PROTOBUF OFF`
- `PROTOBUF_UPDATE_FILES ON`